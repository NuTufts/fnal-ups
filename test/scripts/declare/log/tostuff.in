tostuff/info/info.file

tostuff/catman/cat7:
genlock.7

tostuff/catman/cat8:
eehoor.8

tostuff/man/man6:
poohbear.6

tostuff/man/man8:
sudo.8
