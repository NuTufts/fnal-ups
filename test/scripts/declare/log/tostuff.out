tostuff/info/info.file

tostuff/catman/cat1:
gencat.1

tostuff/catman/cat2:
gendit.2

tostuff/man/man8:
bootpd.8
telnetd.8
